### ISSUE TITLE
Composer identity issue just says command succeeded.

### ISSUE THREAD

**Issue by davidkel** [hyperledger/composer#2652]
It would be good to say card (fully qualified path of created file) with cardname yyy created successfully.

**Comment by mbwhite** [comment]
Updated output
```
$ composer identity issue --card admin@basic-sample-network -u id3 -a "resource:org.acme.sample.SampleParticipant#bob"                             
Issue identity and create Network Card for: id3

✔ Issuing identity. This may take a few seconds...

Successfully created business network card file to 
	Output file: id3@basic-sample-network.card

Command succeeded

$ composer identity issue --card admin@basic-sample-network -u id3 -a "resource:org.acme.sample.SampleParticipant#bob"                        
Issue identity and create Network Card for: id3

✖ Issuing identity. This may take a few seconds...
Error: fabric-ca request register failed with errors [[{"code":0,"message":"Identity 'id3' is already registered"}]]
Command failed


```

**Comment by mbwhite** [comment]
Resolved in PR #2692
